docs/content/Contribution guidelines/operations/packaging-and-versioning.md

```diff
@@ -137,9 +137,8 @@ KubeStellar. According to homebrew convention, users refer to it in
 homebrew as `kubestellar/kubestellar` (e.g., `brew tap
 kubestellar/kubestellar` connects directly to this GitHub repository).
 
-This HomeBrew repository has two formulae.  One called `kcp_cli` has
-the kubectl plugins of kcp. Release v0.11.0 of kcp is always used. A
-formula called `kubestellar_cli` has contents that are needed by a
+This HomeBrew repository has two formulae.  One to control the space
```

Comment:
I think it is better to leave the specificity here, since the brew formula in question really is specific and the point of this document is to talk about dependencies so specificity is important here.

Reply:
The brew formula for kcp is not obviated by the change in Helm charting; the brew formula is supplying executables that clients need.

Reply:
I am not adding this reference into our doc.  it is unnecessary

Reply:
We are talking about a reference that is already in the doc. You are insisting on removing it. I disgree with removing it, but think I have more productive things to spend my time on regarding the word count. So, other than this,
/lgtm